docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,520 @@
+---
+title: "Deploying a High Availability Teleport Cluster"
+description: "Deploying a High Availability Teleport Cluster"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure during an incident in
+your Teleport cluster. You should also make it possible to scale your Auth
+Service and Proxy Service as you register more users and resources with
+Teleport. 
+
+In this guide, we will explain the components of a high-availability Teleport
+deployment.
+
+## Overview
+
+A high-availability Teleport cluster revolves around a group of redundant
+`teleport` processes, each of which runs the Auth Service and Proxy Service,
+plus the infrastructure required to support them. 
+
+This includes:
+
+- **A Layer 4 load balancer** to direct traffic from users and services to an
+  available `teleport` process.
+- A **key-value store** for Auth Service state and audit events that all Auth
```

Comment:
I might find a way to add the terms "cluster state" or "backend" to this item, as these terms are commonly used throughout the docs to describe the key-value store.